src/AccordionPanelGroup.jsx

```diff
@@ -0,0 +1,129 @@
+import _ from 'lodash';
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module AccordionPanelGroup
+ */
+class AccordionPanelGroup extends React.Component {
+	constructor(props) {
+		super(props);
+		const activePanelsArr = [];
+
+		this.accordionPanels = this.props.accordionPanels.map((accordionPanel, i) => {
+			if(accordionPanel.props.isOpen) {
+				activePanelsArr.push(i);
+			}
+			return activePanelsArr;
+		});
```

Comment:
I don't think this `map` call is doing what you expect it to. `this.accordionPanels` will end up being an array that is the length of `this.props.accordionPanels`, where each element is a reference to the same `activePanelsArr` array. You actually never _read_ from `this.accordionPanels` (it gets re-assigned at the top of `renderAccordionPanels`), so I think you are intending to just `reduce` `this.props.accordionPanels` into an `activePanelsArr`

```js
const activePanelsArr = this.props.accodionPanels.reduce((panels, panel, i) => {
  if (panel.props.isOpen) {
    panels.push(i);
  }
  return panels;
}, []);
```

That said, it looks like you're actually storing an array of _indexes_ into the active panels, not the panels themselves, so the `activePanelsArr` naming is confusing. I think the show/hide logic is getting a little convoluted, though, so I'll leave a separate comment about options for cleaning it up.